### PR TITLE
sdk 50 NTRN-230

### PR DESCRIPTION
**Pull request by quasisamurai**

### CODE REVIEW

**Comment by pr0n00gler** [comment]
Please change the target branch to feat/sdk-50 and fix linter issues

File: x/tokenfactory/keeper/msg_server.go [hunk 1]
```diff
@@ -157,7 +157,8 @@ func (server msgServer) ChangeAdmin(goCtx context.Context, msg *types.MsgChangeA
 	}
 
 	if msg.Sender != authorityMetadata.GetAdmin() {
-		return nil, types.ErrUnauthorized
+		return nil, types.ErrDenomDoesNotExist.Wrapf("need: %s, received: %s, denom: %s", authorityMetadata.GetAdmin(), msg.Sender, msg.Denom)
```
Comment (mmikev) [inline x/tokenfactory/keeper/msg_server.go]: Huh?
The error doesn't seem right

File: x/tokenfactory/types/msgs.go [hunk 2]
```diff
@@ -32,16 +32,6 @@ func NewMsgCreateDenom(sender, subdenom string) *MsgCreateDenom {
...
-	_, err := sdk.AccAddressFromBech32(m.Sender)
-	if err != nil {
-		return errorsmod.Wrapf(sdkerrors.ErrInvalidAddress, "Invalid sender address (%s)", err)
-	}
-
-	_, err = GetTokenDenom(m.Sender, m.Subdenom)
-	if err != nil {
-		return errorsmod.Wrap(ErrInvalidDenom, err.Error())
-	}
-
```
Comment (mmikev) [inline x/tokenfactory/types/msgs.go]: Why it is removed?

File: x/tokenfactory/types/msgs.go
Comment (mmikev) [inline x/tokenfactory/types/msgs.go:1]: What's the point of the changes here?

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**